mssqlcli/mssql_cli.py

```diff
@@ -340,14 +331,11 @@ def _execute_interactive_command(self, text):
 
         except NotImplementedError:
             click.secho('Not Yet Implemented.', fg="yellow")
-        except Exception as e:
```

Comment:
Again, where has generic error handling moved to? Is it still cause somewhere in the stack?

Reply:
This was removed due to PEP 8 guidelines. I believe that the function has all exceptions accounted for already, but I am unsure here.

I'm not exactly tied down to doing everything PEP 8 recommends—I think some flexibility is ok, maybe we just add this back. What are your thoughts @kevcunnane?